packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -2749,16 +2763,31 @@ export const components = ({
           [`.${tableBodyClasses.root} &`]: {
             // Target is 48px height
             paddingBlock: odysseyTokens.Spacing3,
+
+            [`& .${tableCellClasses.root}::after`]: {
+              top: `-${odysseyTokens.Spacing3} !important`,
```

Comment:
@jordankoschei-okta This comment applies to a few of these properties, and is just a general question... Should we be using the logical properties for positioning like these?
```css
insetBlockStart: `-${odysseyTokens.Spacing3} !important`
```
I'm assuming in this scenario we may need to override their styling explicitly though

Reply:
I agree, but let's tackle this as a separate PR since it impacts beyond the scope of this one.